cloud/README.md

```diff
@@ -0,0 +1,65 @@
+# Please do not publish a solution in either a fork or clone. Keep all of your work local!
+#### If you do want to store your work, we ask that you keep it in a private repo (github lets everyone do this now).
+
+## Introduction
+
+This project is designed to get you familiar with some basic functionalities of our ArchFX Cloud product.
+
+Arch has built a complete framework to help factories monitoring their production lines. Arch FX Cloud is the core of
+this product, as it is the main user interface but also the link with all our other products.
+
+It is basically a webapp served by a Django server, with all the needed backend tools.
+
+To test the skills you will need to be part of the Cloud team, we will build a small API running with Django.
+
+*It should take less than 1 hour to complete. If you think some parts need more, please tell us.*
+
+## Description
+
+The goal of this exercise is to build an API that will allow a user to register time blocks with a name
+and then query the resulting timeline.
+
+1. Setup a Django server within a Docker container. This server will need a PostgreSQL database to store models, which
+   will also be into a Docker container. We recommend describing these containers using docker-compose.
+2. Define the `Timeblock` model and migrations to save it in DB.
```

Comment:
Similarly, do you think it makes sense to provide the template/boilerplate Django project folder (like `server/*`)? Perhaps even include the `Timeblock` model stub where you expect it to be. 

The motivation behind these comments is to give you a more "standard" solution to look at. One thing that I often find helps candidates as well is to provide them a basic set of unit tests that they can run to make sure their solution covers "the basics". I have my own private larger test suite that exercises corner cases and catches errors in algorithms, but that depends on a consistent structure and input/output to the system.